spaceship/lib/spaceship/tunes/build.rb

```diff
@@ -173,29 +174,11 @@ def submit_for_beta_review!(metadata)
           build_number: self.build_version,
           platform: self.platform,
 
-          # Required Metadata:
-          changelog: "No changelog provided",
-          description: "No app description provided",
-          feedback_email: "contact@company.com",
-          marketing_url: "http://marketing.com",
-          first_name: "Felix",
-          last_name: "Krause",
-          review_email: "contact@company.com",
-          phone_number: "0123456789",
-          significant_change: false,
-
-          # Optional Metadata:
-          privacy_policy_url: nil,
-          review_user_name: nil,
-          review_password: nil,
-          encryption: false
+          review_notes: "Not provided" # this seems to be required, even though it is specified as optional
```

Comment:
Turns out a lot of this is required, I was getting review_notes as an error because it as the only field that wasn't preopulated, I think I'll remove this default value.